packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -529,9 +529,6 @@ - (FLTTextureMessage *)onPlayerSetup:(FLTVideoPlayer *)player
 }
 
 - (void)initialize:(FlutterError *__autoreleasing *)error {
-  // Allow audio playback when the Ring/Silent switch is set to silent
-  [[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryPlayback error:nil];
```

Comment:
This is a breaking change: if someone updated `video_player_avfoundation` but not `video_player`, it would suddenly stop doing setup that it used to do.

This is why [my comment](https://github.com/flutter/plugins/pull/5872#issuecomment-1145229349) described adding a new init to the platform as well. The existing platform interface init method needs to continue working as it has before; the new one can skip this. (Looking at that comment again, I don't think it needs the parameter I described; it should be able to unconditionally not do the setup, since anyone calling the new init will also be calling the new method when necessary.)